requirements.txt

```diff
@@ -59,12 +59,12 @@ psycopg2==2.6.2 \
     --hash=sha256:8ffbd1128df23c9fdfc3499084021055b3df7818f12ef87af5b3f33e27d58b0a \
     --hash=sha256:8c3b69d743e408527208d5ed6aa136b821bbd3cb1e236aa8479ff47ea986769c \
     --hash=sha256:70490e12ed9c5c818ecd85d185d363335cc8a8cbf7212e3c185431c79ff8c05c
-elasticsearch==1.9.0 \
-    --hash=sha256:cbee23c29f9947b1b78be5c0f741c1f8b504d3c53163b1355ea08f69cbd0fc0c \
-    --hash=sha256:eb0d4b5b20a4ccf9e49bd20498b6215d5e960d8a63a0eae44cd4431e3f303a24
-elasticsearch-dsl==0.0.11 \
-    --hash=sha256:663fb62ad39200c7d903e973aa0aa693578613264d83796455cbf4cd172bd878 \
-    --hash=sha256:59a76c4142478a1952bba6f9a9ca4fc7b029afb619e8ffcf0d135ce37ea692da
+elasticsearch==5.1.0 \
+    --hash=sha256:ee95918958508fd34e533f78d0790e0e5ee03d0942b8988d34e6b6c4467edf66 \
+    --hash=sha256:a0bc4508764aa14c0cd15840a0ab38020b68e17d699274417986e3da0cee82fa
+elasticsearch-dsl==5.1.0 \
```

Comment:
There is now a `5.2.0` of both `elasticsearch` and `elasticsearch-dsl` in pypi. Is now maybe a good time to push to that too?